bindings/huawei/obs/obs_service.go

```diff
@@ -31,22 +35,22 @@ type HuaweiOBSService struct {
 	client *obs.ObsClient
 }
 
-func (s *HuaweiOBSService) PutObject(input *obs.PutObjectInput) (output *obs.PutObjectOutput, err error) {
+func (s *HuaweiOBSService) PutObject(ctx context.Context, input *obs.PutObjectInput) (output *obs.PutObjectOutput, err error) {
 	return s.client.PutObject(input)
```

Comment:
You need to add a second argument `obs.WithRequestContext(ctx)`. See https://github.com/ItalyPaleAle/dapr-components-contrib/blob/bindings-context/bindings/huawei/obs/obs_service.go#L39

Reply:
Hah, thanks.